src/index.js

```diff
@@ -178,8 +178,14 @@ export default class DataLoader {
   // Internal
   
   _modelImport(model_id, results, opts) {
-    if(this.app.get('data-loader').listenerCount('models.'+model_id) == 0) {
+    let loader = this.app.get('data-loader')
+    //stub default handlers on "model" and "models" events
+    if((loader.listenerCount('models.'+model_id)) == 0) {
```

Comment:
@ScottMaxson this will still cause the 'after' case to create duplicate entries though right?  If I have `app.get('data-loader').after('models.my-model')` the above test will fail and I'll get two callbacks, e.g. double entries?